sub-packages/bionemo-geneformer/src/bionemo/geneformer/api.py

```diff
@@ -34,6 +35,18 @@
 GeneformerModel = MegatronBioBertModel
 
 
+class BERTMLMLossWithReductionNoForward(BERTMLMLossWithReduction):
```

Comment:
In the future, we need to add a mechanism to pass options into a configurable `LossReductionClass`. Right now we have to override the class/use the default.